x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/summary_step/components/post_save_options/post_save_options.tsx

```diff
@@ -79,7 +79,7 @@ export const PostSaveOptions: FC<Props> = ({ jobRunner }) => {
             datafeedState === DATAFEED_STATE.STARTING || datafeedState === DATAFEED_STATE.STARTED
           }
           onClick={startJobInRealTime}
-          data-test-subj="mlButtonUseFullData3"
+          data-test-subj="mlJobWizardButtonRunInRealtime"
```

Comment:
Nit - but elsewhere we use `RealTime`, so should be `mlJobWizardButtonRunInRealTime` for consistency.

Reply:
Changed in c3ef36f